экзамен/экзамен/Program.cs

```diff
@@ -0,0 +1,17 @@
+namespace экзамен
+{
+    internal static class Program
+    {
+        /// <summary>
+        ///  The main entry point for the application.
+        /// </summary>
+        [STAThread]
+        static void Main()
+        {
+            // To customize application configuration such as set high DPI settings or default font,
+            // see https://aka.ms/applicationconfiguration.
+            ApplicationConfiguration.Initialize();
```

Comment:
Для .NET Core приложение с графическим интерфейсом конфигурируется не так. Вы, видимо, взяли не тот шаблон.
```suggestion
            Application.SetHighDpiMode(HighDpiMode.SystemAware);
            Application.EnableVisualStyles();
            Application.SetCompatibleTextRenderingDefault(false);
```